data/simpy.gram

```diff
@@ -24,11 +24,96 @@ small_stmt[stmt_ty]:
 compound_stmt[stmt_ty]: if_stmt | while_stmt | for_stmt | with_stmt | try_stmt | function_def | class_def
 
 # NOTE: yield_expression may start with 'yield'; yield_expr must start with 'yield'
-assignment: 
-    | target ':' expression ['=' yield_expression]
-    | (star_targets '=')+ (yield_expr | expressions)
-    | target augassign (yield_expr | expressions)
-augassign: ('+=' | '-=' | '*=' | '@=' | '/=' | '%=' | '&=' | '|=' | '^=' | '<<=' | '>>=' | '**=' | '//=')
+assignment:
+    | a=target ':' b=expression '=' c=yield_expression {
+         _Py_AnnAssign(SET_CONTEXT(a, Store), b, c, 1,
+         EXTRA(a, expr_type, c, expr_type)) }
+    | a=target ':' b=expression {
+         _Py_AnnAssign(SET_CONTEXT(a, Store), b, NULL, 1,
+         EXTRA(a, expr_type, b, expr_type)) }
+    | a=target '=' b=yield_expression {
+         _Py_Assign(singleton_seq(p, SET_CONTEXT(a, Store)), b, NULL,
+         EXTRA(a, expr_type, b, expr_type)) }
+    | (star_targets '=')+ yield_expr
+    | (star_targets '=')+ expressions
+    | a=target '+=' b=yield_expr {
```

Comment:
We need a way to allow integer values as the types for rules because this is a bit ridiculous.....